test/e2e/framework/helpers/yunikorn/wrappers.go

```diff
@@ -106,8 +106,6 @@ func UpdateCustomConfigMapWrapper(oldConfigMap *v1.ConfigMap, schedPolicy string
 	Ω(err3).NotTo(HaveOccurred())
 	Ω(d).NotTo(BeNil())
 
-	// Updating scheduler pod annotation to trigger force refresh of configmaps
-	Ω(k.UpdateYunikornSchedulerPodAnnotation(annotation)).NotTo(HaveOccurred())
```

Comment:
The `KubeCtl.UpdateYunikornSchedulerPodAnnotation` is only used here. We can also remove it. Thanks.

https://github.com/apache/yunikorn-k8shim/blob/292e26fd9d910be39bb98f3bc01b7445179dff9e/test/e2e/framework/helpers/k8s/k8s_utils.go#L927

Reply:
yes the definition of this function should be removed too `UpdateYunikornSchedulerPodAnnotation()`

Reply:
Thanks, the latest PR rev has this suggested change.